### DISCUSSION TITLE
Clear DF bit for packets going through IPSec tunnel

### DISCUSSION THREAD

**Discussion by aleemgsl** [strongswan/strongswan#1460] [Q&A] [answered]
Hi All,

We have an IPSec tunnel setup between two pod networks. I would like to clear DF bit in the packets traversing through IPSec tunnel. Can you please share the steps to do the same?

**Comment by tobiasbrunner** [comment, marked as answer]
You can try disabling the `copy_df` option for the child config in [swanctl.conf](https://docs.strongswan.org/docs/5.9/swanctl/swanctlConf.html#_connections_conn_children).

**Reply by aleemgsl** [reply]
Thanks.

We are using ipsec.conf. 

root@123:~# ipsec --version
Linux strongSwan U5.6.2/K5.4.0-132-generic
Institute for Internet Technologies and Applications
University of Applied Sciences Rapperswil, Switzerland

Can you please share which parameter to be added in ipsec.conf?

**Reply by tobiasbrunner** [reply]
There is none that controls this for the deprecated ipsec.conf/stroke interface, you have to use swanctl.conf/vici. However, the option was only added with 5.7.0, so that version is too old anyway.

**Reply by aleemgsl** [reply]
Thanks. Let me use swanctl.conf. 

I am using ubuntu 18.04. Is it supported, or do I need to upgrade to Ubuntu 20.04?

**Reply by tobiasbrunner** [reply]
You clearly do as 5.6.2 < 5.7.0. What exactly is your goal regarding DF bit anyway?

**Reply by aleemgsl** [reply]
Thanks tobiasbrunner. 

SCTP traffic going through the IPSec tunnel was getting dropped at the remote site. And hence we wanted to check if clearing the DF bit will resolve it or not. It turned out to be a bad checksum issue.
